pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -30,6 +30,8 @@ import (
 
 	"gopkg.in/gcfg.v1"
 
+	"golang.org/x/net/context"
```

Comment:
Thank you for reviewing this. I pushed this as a WIP and used it as a starting point for [CloudProvider metrics](https://github.com/kubernetes/features/issues/182) proposal. I will clean it up and add more checks for nil and stuff. 

I will give you another shout when it is ready for review.